pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -2146,8 +2146,12 @@ type StorageScopeSpec struct {
 	// +kubebuilder:pruning:PreserveUnknownFields
 	// +nullable
 	// +optional
-	Config    map[string]string `json:"config,omitempty"`
-	Selection `json:",inline"`
+	Config map[string]string `json:"config,omitempty"`
+	// ResourceClasses are the list of different resources
+	// +nullable
+	// +optional
+	ResourceClasses []ResourceClass `json:"resourceClasses,omitempty"`
```

Comment:
Can we just define this directly as a map? Then no need to define our own `ResourceClass` type or convert it to a map in code.
```suggestion
	ResourceClasses map[string]v1.ResourceRequirements `json:"resourceClasses,omitempty"`
```

Reply:
Thank's, updated like this.